package.json

```diff
@@ -195,6 +215,21 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.show",
+        "title": "Show Metadata",
+        "category": "ONE"
+      },
+      {
+        "command": "one.viewer.relation.show",
```

Comment:
Could you remove these `Relation Viewer`-related lines? (2)

Reply:
All right, I'll apply that part.